mypy/build.py

```diff
@@ -871,11 +875,11 @@ def find_cache_meta(id: str, path: str, manager: BuildManager) -> Optional[Cache
     m = CacheMeta(
         meta.get('id'),
         meta.get('path'),
-        meta.get('mtime'),
+        int(meta.get('mtime')) if 'mtime' in meta else None,
```

Comment:
Use `meta['mtime']` instead f `meta.get('mtime')` as the latter would conflict with strict optional checking.